action.yml

```diff
@@ -116,6 +131,9 @@ inputs:
   blackduck_url:
     description: 'URL for blackduck hub'
     required: false
+  blackduck_apiToken:
```

Comment:
why this is added? this is an old parameter i guess.

Reply:
Nice catch. I forgot that blackduck_apiToken was deprecated. I was getting unexpected param.
Removing it.